build.sbt

```diff
@@ -56,6 +56,24 @@ lazy val commonDependencies = Seq(
   "org.apache.commons" % "commons-csv"        % "1.10.0"
 )
 
+lazy val commonSettings = Seq(
+  javaOptions ++= Seq(
+    "--add-opens=java.base/java.lang=ALL-UNNAMED",
+    "--add-opens=java.base/java.util=ALL-UNNAMED",
+    "--add-opens=java.base/java.io=ALL-UNNAMED"
+  ),
+  Test / javaOptions ++= Seq(
+    "--add-opens=java.base/java.lang=ALL-UNNAMED",
+    "--add-opens=java.base/java.util=ALL-UNNAMED",
+    "--add-opens=java.base/java.io=ALL-UNNAMED"
+  ),
+  run / javaOptions ++= Seq(
+    "--add-opens=java.base/java.lang=ALL-UNNAMED",
+    "--add-opens=java.base/java.util=ALL-UNNAMED",
+    "--add-opens=java.base/java.io=ALL-UNNAMED"
+  )
+)
+
 lazy val buil1dSettings = Defaults.coreDefaultSettings ++ Seq(
```

Comment:
I found this buildSettings in line 23, and it would be duplicated if we changed the name to the same.
https://github.com/neuvector/manager/blob/899aa48eb1dbf27cc8a49a76b44b1da3d9e0961c/build.sbt#L23

Also, buil1dSettings seems unused after check. 
@BinX-Suse Could you help with checking on this? I think we can remove it if we don't need it, thank you!
https://github.com/neuvector/manager/blob/899aa48eb1dbf27cc8a49a76b44b1da3d9e0961c/build.sbt#L77

Reply:
I will remove it